src/Controls/src/Core.Design/ButtonContentDesignTypeConverter.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.ComponentModel;
+
+namespace Microsoft.Maui.Controls.Design
+{
+	public class ButtonContentDesignTypeConverter : StringConverter
+	{
+		private enum ImagePosition { Left, Top, Right, Bottom }; // MUST MATCH values of ButtonContentConverter.ImagePosition
```

Comment:
This one feels like you could use `HashSet<string>` w/ appropriate StringComparer instead of defining an enum and `Enum.Parse()`.

Reply:
Ok that seems like a good reason to use it. 👍

Reply:
BTW I found that a) runtime converters do not always handle well corner cases and b) don't match build time converters. For example, runtime CornerRadiusTypeConverter allows "5,foo,bar" (creates uniform radius) while build time converted does not allow that.